src/ResourceManager/IotCentral/Commands.IotCentral/Az.IotCentral.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
```

Comment:
Is there any reason these module versions are different?

Reply:
Differs depending on .NetFramework vs. .NetCore